### DISCUSSION TITLE
Welcome to signals-srkw Discussions!

### DISCUSSION THREAD

**Discussion by scottveirs** [orcasound/signals-srkw#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by UXBrendan** [comment]
I hope this is the appropriate place for this discussion, but I wanted to bring your attention to a SRKW call catalog that has been redesigned by designers of the Orcasound UX team. 

https://drive.google.com/file/d/172LSUJk1hIicYMTL74x-usCl5P3ydzy7/view?usp=sharing

**Reply by scottveirs** [reply]
Hey @UXBrendan , after my post-standup discussion with Rachael and DaveB, I discovered this old thread.

I do think this public repo is a great place to solicit feedback on current/future versions of SRKW signal catalogs!

I'd suggest creating a new discussion topic for the Orcasound version of a new call catalog. You should be able to place the PNG directly in the start of the Discussion topic. Then it will be archived in Github (in case the Gdrive link breaks some day) and you can pose some questions about the UI/design that references the PNG.

To put the PNG in the discussion start (or any reply on Github discussions or issues), you can drag/drop into the compose window, or select the "Paste, drop, or click to add files" icon/text at the bottom of the compose window (gray highlight here):

<img width="947" alt="Screenshot 2024-09-25 at 12 22 00 PM" src="https://github.com/user-attachments/assets/5f16da98-9e90-493b-852b-3f63fbc0b38b">

Actually, I just tried with your 17MB PNG and Github complained about it being >10MB, so maybe you'll have to post a low-res version (or sub-sections of the design with features about which you want feedback), like this (<1MB version):

<img width="415" alt="Screenshot 2024-09-25 at 12 22 54 PM" src="https://github.com/user-attachments/assets/6ce63a81-968c-4aba-94f3-a150db10e582">

**Reply by UXBrendan** [reply]
@scottveirs 

Thanks! Earlier this year or maybe at the end of last year, I had Sarah Loup prototype the call catalog. I can upload a video of a demo if that helps, since there are interactive animations involved. I'm really excited about this design, and would love to see it developed! Do you think we could recruit some volunteer devs to work on it?
